### DISCUSSION TITLE
The Name is strange

### DISCUSSION THREAD

**Discussion by heho** [microsoft/griffel#69] [Q&A] [answered]
Now this might be a strange kind of issue, but given that you are using a German word as a name and German is my mother tongue, I feel qualified to comment. Whilst Griffel definitely means "stylus" this meaning is out of style. The kind of stylus called Griffel has not been in use for several decades, and so next to no one uses that word in that sense. It is however still a used word, but mainly in a derogatory sense, to describe the fingers of a person. Usually when that Person touches something they should not be touching, in the worst case another Person.

**Comment by miroslavstastny** [comment, marked as answer]
Naming is hard 😃  We definitely do not want to offend anybody, we also do not expect engineers to see any deeper sense in the name itself. Languages evolve, what seems to be a good name today might be a sensitive topic tomorrow.
The most important thing for us is to develop a library which serves its purpose.
Thanks for your understanding.

**Reply by heho** [reply]
It definitely is. I'm pretty sure no one is going to be offended, especially since you stated the etymology in the README.
